crates/burrego/src/opa/builtins/regex.rs

```diff
@@ -117,7 +118,7 @@ impl Display for ExpressionList {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let mut result = String::new();
```

Comment:
do we really need this intermediate String? Can we just write in the formatter inside the loop? something like:

```
fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        for expression in self.0.iter() {
            write!(f, "{}", expression)?;
        }
        Ok(())
    }
```
Then we wouldn't need this line ` use std::fmt::Write as _; // import without risk of name clashing`

Reply:
:+1: with @raulcabello . If you want we can also tackle that with another PR, I'm fine merging that with the clippy warning about burrego

Reply:
agh, thanks, true! done in https://github.com/kubewarden/policy-evaluator/pull/146/commits/03c8a36ebbb30a7568957d1b592510af1b2e2297